### DISCUSSION TITLE
Deployment of nextjs app using .exe file

### DISCUSSION THREAD

**Discussion by BharathKS77** [vercel/next.js#40934] [Help]
Hi ,

We have a requirement to deploy nextjs app into a windows machine .So we need to generate a .exe file out of the project .Can we please get guidance regarding this .

**Comment by icyJoseph** [comment]
I think you can use: https://github.com/vercel/next.js/tree/canary/examples/with-electron-typescript, as a starting point. 

Add `name` and `version` to the `package.json` file, and then `yarn pack-app` or `npm run pack-app` should build an executable for your system. I've done it on Mac though, can't vouch for Windows, but you can give it a go.

Otherwise, Tauri is also an option worth exploring, https://tauri.app/v1/guides/building/windows/.

**Comment by JaosnHsieh** [comment]
Old articles but it was woking on next.js v8

https://medium.com/@evenchange4/deploy-a-commercial-next-js-application-with-pkg-and-docker-5c73d4af2ee

https://github.com/evenchange4/nextjs-pkg-docker-alpine
